File: app/src/main/java/co/tpcreative/supersafe/model/HelpAndSupport.java
package co.tpcreative.supersafe.model;
import java.io.Serializable;

public class HelpAndSupport implements Serializable{

    public Categories categories;
    public String title;
    public String content;
    public String nummberName;

    public HelpAndSupport(Categories categories,String title,String content,String nummberName){
        this.categories = categories;
        this.title = title;
        this.content = content;
        this.nummberName = nummberName;
    }
    public HelpAndSupport(){
        this.categories = new Categories();
        this.title = "";
        this.content = "";
        this.nummberName = null;
    }

    public int getCategoryId() {
        return categories.id;
    }

    public String getCategoryName() {
        return categories.name;
    }

}

File: app/src/main/java/co/tpcreative/supersafe/model/HelpAndSupport.kt
package co.tpcreative.supersafe.model
import java.io.Serializable

class HelpAndSupport : Serializable {
    var categories: Categories?
    var title: String?
    var content: String?
    var nummberName: String?

    constructor(categories: Categories?, title: String?, content: String?, nummberName: String?) {
        this.categories = categories
        this.title = title
        this.content = content
        this.nummberName = nummberName
    }

    constructor() {
        categories = Categories()
        title = ""
        content = ""
        nummberName = null
    }

    fun getCategoryId(): Int? {
        return categories?.id
    }

    fun getCategoryName(): String? {
        return categories?.name
    }
}